### DISCUSSION TITLE
KeyCloak SSO Integration with AWS Organizations

### DISCUSSION THREAD

**Discussion by KCol13** [keycloak/keycloak#28254] [Q&A]
Is there a way to integrate KeyCloak with an AWS Organization to facilitate giving users SSO access to multiple accounts. 

Currently the only way to set up multi account access that I'm aware of is to set KeyCloak as an Identity Provider in each AWS account, create roles in each AWS account, then map all of those roles to users/groups in KeyCloak, but I'm wondering if there's a better way to integrate KeyCloak with an AWS Organization?

**Comment by codespearhead** [comment]
What do you mean by "give users SSO access to multiple accounts"? I'm not sure I understand the use case.
